be/src/runtime/memory/lru_cache_policy.h

```diff
@@ -84,26 +76,24 @@ class LRUCachePolicy : public CachePolicy {
 
     // Try to prune the cache if expired.
     void prune_stale() override {
+        COUNTER_SET(_freed_entrys_counter, (int64_t)0);
```

Comment:
use 0LL replace （int64_t) 0

Reply:
0L failed to compile on mac, let me try 0LL

Reply:
warning: Call to member function 'set' is ambiguous

virtual void set(int64_t value)
virtual void set(double value)